Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -251,6 +255,8 @@ extension _ChatChannelListVC: _ChatChannelListControllerDelegate {
                         collectionView.reloadItems(at: [index])
                     }
                 }
+                
+                channelsCount = controller.channels.count
```

Comment:
I think the `count` value should be obtained outside of the closure. Also, do we need to somehow handle the edge case where `controller.channel.count` < `channelsCount`?

Reply:
> I think the `count` value should be obtained outside of the closure.

I think it should not be necessary as the closure is non-escaping so performed immediately and synchronously, but just to be sure we can do that.

> Also, do we need to somehow handle the edge case where `controller.channel.count` < `channelsCount`?

I don't see any reason why, do you?